include/klee/Expr.h

```diff
@@ -449,33 +450,33 @@ class NotOptimizedExpr : public NonConstantExpr {
 
 /// Class representing a byte update of an array.
 class UpdateNode {
-  friend class UpdateList;  
+  friend class UpdateList;
 
-  mutable unsigned refCount;
   // cache instead of recalc
   unsigned hashValue;
 
 public:
-  const UpdateNode *next;
+  const ref<UpdateNode> next;
   ref<Expr> index, value;
-  
+
+  /// @brief Required by klee::ref-managed objects
+  mutable struct ReferenceCounter __refCount;
```

Comment:
As you changed `ReferenceCounter` to be a `class`, I would suggest you change it here and in the other places (5 in total).

Reply:
@jbuening Thanks for spotting this! Fixed.